db/controllers/album-controller.js

```diff
@@ -146,4 +148,35 @@ albumRouter.get("/search/:query", async (req, res) => {
     });
 });
 
+// Get all albums currently stored in our database
+albumRouter.get("/", async (req, res) => {
+    // Get all reviews
+    Album.findAll({
+        include: [
+            {
+                model: Like,
+                attributes: [],
+                duplicating: false,
+            },
+        ],
+        attributes: [
+            "id",
+            "spotifyId",
+            "image",
+            "name",
+            "artist",
+            [sequelize.fn("COUNT", sequelize.col("Likes.AlbumId")), "likesCount"],
```

Comment:
This includes the number of likes the album has with the response

Reply:
very nice

Reply:
looks good!